.github/workflows/build.yml

```diff
@@ -21,118 +21,52 @@ jobs:
   # Test the compilation compatibility
   linux_build:
     name: ${{ matrix.config.name }}
-    runs-on: ${{ matrix.config.os }}
```

Comment:
I suggest you ignore following paths for `linux_build`
```
openfpga_flow/**
docs/**
docker/**
.github/**
```
These directory triggers unnecessary build

Reply:
I am not sure how you can skip just one of the job from `linux_build` and `regression_tests`. There is options `if` for each job. But skipping entire workflow is easier. 
I think the better implementation will be to separate `linux_build` and `regression_tests` in two different workflows.

Reply:
``openfpga_flow`` can not be ignored as we keep adding new tests
``docs`` cannot be ignored because documentation build is part of CI
``docker`` image is actually used in CI as well.
``github`` is where CI tests is located.

In the end, I do not want to see a trash corner in our repo which can bypass CI

Reply:
Any change in `openfpga_flow` wont require `linux_build` task to run, just `regression_test` will run.
same is true for `github`, `docker` and `doc` none of it requires testing `linux_build`.
Note: Regression test will still run in those case.